<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>face_recognitionClass</class>
 <widget class="QMainWindow" name="face_recognitionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>face_recognition</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1001</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home_0">
     <widget class="QLabel" name="h_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>551</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;BRL RGB_D人脸识别系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="h_RGBpic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>261</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="h_Dpic">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>120</y>
        <width>261</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>120</y>
        <width>81</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="h_name">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;姓名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="h_id">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;学号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="h_res">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="enroll_button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>注 册</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_button">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>440</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退 出</string>
      </property>
     </widget>
     <widget class="QLabel" name="h_threshold">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>530</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>阈 值</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="h_th">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>530</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="h_name_label">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>140</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="h_id_label">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>200</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>270</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="search_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查 询</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="enroll_1">
     <widget class="QLabel" name="en_RGBpic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_Dpic">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_RGB_save">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>NO RGB-PICTURE CAPTURED</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_D_save">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>300</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>NO DEPTH-PICTURE CAPTURED</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_note">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>70</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;注意：&lt;/p&gt;&lt;p&gt;1.点击拍照按钮获取注册图像，显示在下方。&lt;/p&gt;&lt;p&gt;2.在对应框中输入姓名和编号。&lt;/p&gt;&lt;p&gt;3.当确认无误后点击拍照按钮完成注册。&lt;/p&gt;&lt;p&gt;4.点击返回主界面按钮，返回上一界面。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_name_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>270</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;姓名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_sno_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>320</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;学号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="en_name_edit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="en_sno_edit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>320</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="en_capturebutton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>370</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>拍 照</string>
      </property>
     </widget>
     <widget class="QPushButton" name="en_homebutton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>420</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回主页面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="en_exitbutton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>470</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退 出</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_group_lable">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>370</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;组织&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="en_groupBox">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="en_male">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>420</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>男</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="en_female">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>420</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>女</string>
      </property>
     </widget>
     <widget class="QLabel" name="en_gender_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;性别&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="en_grade">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>470</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>470</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;年级&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="search_2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>981</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sr_name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sr_name_lb">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>31</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;姓名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="sr_time_lb">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>31</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="sr_result_lb">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>31</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sr_result">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>31</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>31</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sr_searchButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>31</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查 询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sr_homebutton">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>31</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="face_recognition.qrc"/>
 </resources>
 <connections/>
</ui>
